flattentool/json_input.py

```diff
@@ -60,6 +64,15 @@ def __init__(self, json_filename=None, root_json_dict=None, schema_parser=None,
         else:
             self.rollup = False
 
+        if self.xml:
+            with codecs.open(json_filename, 'rb') as xml_file:
+                root_json_dict = xmltodict.parse(
+                    xml_file,
+                    force_list=(root_list_path,),
+                    force_cdata=True,
+                    )['iati-activities']
```

Comment:
We are hardcoding iati-activites here which should really be configurable or potentially default to the last item in root_list_path.

Also If there are no iati-activites at all does force list create a key? or this will cause a 500.